config/application.rb

```diff
@@ -71,6 +71,7 @@ class Application < Rails::Application
     config.action_mailer.delivery_method = :smtp
     config.action_mailer.perform_deliveries = true
     config.action_mailer.raise_delivery_errors = true
+    config.action_mailer.deliver_later_queue_name = :soon
```

Comment:
🤔 Apparently Rails only added this in 5.x, so I'll need to figure out an alternative before merge